src/lib/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+
+type BadgeSize = "small" | "large";
+
+export interface BadgeProps {
+  icon: string;
+  alt: string;
```

Comment:
I think the image does not need alt text in this case, or if you want to keep it lets create a separate prop called `ImageProps` as in MUI.